src/App.tsx

```diff
@@ -15,36 +23,99 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App: React.FC = () => {
+enum SortBy {
+  'default',
+  'alpabet',
+  'length',
+}
+
+function getGoods(goodsFromServList: string[]): GoodsList {
+  return goodsFromServList.map((str, index) => ({ id: index, val: str }));
+}
+
+function prepGoods(goods: GoodsList, oper: SortBy, revOp: boolean): GoodsList {
+  let tempArr = getGoods(goodsFromServer);
+
+  if (oper === 0) {
+    return revOp ? tempArr.reverse() : tempArr;
```

Comment:
The `oper` parameter is compared to a raw number which makes the code less readable. Use the `SortBy` enum values for comparisons to improve code clarity and maintainability.